src/boolean_result.cpp

```diff
@@ -658,6 +658,39 @@ void CreateProperties(Manifold::Impl &outR, const Manifold::Impl &inP,
     }
   }
 }
+
+void ReorderHalfedges(VecView<Halfedge> &halfedges) {
+  // halfedges in the same face are added in non-deterministic order, so we have
+  // to reorder them for determinism
```

Comment:
There's a `CreateHalfedges` call at the end of `Face2Tri` - would it be more efficient to merge this logic in there?

Reply:
Probably, but `CreateHalfedges` is called in multiple places and this reordering is only concerned with boolean result. I think we can try to optimize this later when this is causing slowness.